Makefile

```diff
@@ -107,4 +107,4 @@ public_repo:
 
 public_perm: public_repo
 	@echo ensure public permission is 0777
-	@chmod -R 0777 public
```

Comment:
Do we need to open the permissions that widely? I thought it'd be enough to have that directory writable by the Rails process. But maybe it doesn't matter when this is just used as a dev env.

Reply:
I was only following the original instructions in the DEVELOPING.md which specified chmod -R 0777 public.

If we only need to ensure that public/repo exists with a specific permission, we can refine this further.